sky/shell/BUILD.gn

```diff
@@ -178,6 +178,8 @@ if (is_android) {
       "//sky/services/activity:activity_lib",
       "//sky/services/activity:interfaces_java",
       "//sky/services/engine:interfaces_java",
+      "//sky/services/gcm:gcm_lib",
+      "//sky/services/gcm:interfaces_java",
```

Comment:
The first step is to not add this dependency to the build.  Instead of linking GCM into SkyShell.apk, we should add it as a dependency to Fitness.apk.